app/src/main/java/com/xa/xpensauditor/LoginActivity.java

```diff
@@ -17,6 +17,9 @@
 import android.widget.ProgressBar;
 import android.widget.Toast;
 
+/**
+ * Landing screen view of the application. Handles login related activites and sets the Firebase instance up.
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 109).